e2-studio-star-rx72n-firmware/src/hardware_init.c

```
@@ -509,6 +528,172 @@ static rx_err_t internal_gpio_init_gptw_pwm(void)
+ * @param[in] port_pin Encoded port/pin value from hardware_config.h
+ * @param[in] initial_high True to drive HIGH on startup, false for LOW
+ *
+ * @pre Pin must have MPC already configured via rx_mpc_set_gpio()
+ * @pre Port base address must be valid for the given pin
+ * @post Pin configured as GPIO output at the requested level
+ *
+ * @note Thread-safe. No shared state modified.
+ *
+ * @since Version 1.2.0
+ */
+static inline void internal_gpio_set_output(rx_port_pin_t port_pin, bool initial_high)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'internal_gpio_set_output' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)